### ISSUE TITLE
Vegas doesn't open configuration dialog for UTVideo 20.4.0+

### ISSUE THREAD

**Issue by tvasenin** [umezawatakeshi/utvideo#43]
Starting from v20.4.0, Vegas doesn't allow to open UTVideo configuration dialog ("Configure..." button is disabled).
The issue is only with the configuration dialog, encoding/decoding itself is working fine.

Version 20.3.0 works fine in Vegas.
VirtualDub2 has no issues with any version.

Seems like the incompatibility was introduced in commit 344e70e893f8439998e863ad9a2f951ad4a111db.
My guess is that Vegas calls the codec incorrectly, and the corresponding check in `VCMCodec` should be relaxed as a workaround.

There's another report for a different application, but the bug is probably the same:
https://forums.guru3d.com/threads/msi-afterburner-external-vfw-codec-recording-not-working.432000/#post-5871279

**Comment by umezawatakeshi** [comment]
I downloaded Vegas 19 free trial (Version 19.0, Build 550) and confirmed the behavior: Vegas doesn't open UtVideo config dialog ("Configure..." button is grayed out) while it can both encode and decode video clips via UtVideo.

I investigated this behavior and concluded that it would be appropriate to address this issue by modifying UtVideo (though it is not necessarily an UtVideo's bug --- This issue is due to the fact that the interpretation of the "mode" flag passed to codecs is not well documented and is ambiguous ).

--

By the way, I also downloaded MSI Afterburner but could not find video recording feature.
